src/xo-app/sr/tab-advanced.js

```diff
@@ -1,37 +1,95 @@
 import _ from 'intl'
+import Component from 'base-component'
 import Copiable from 'copiable'
 import React from 'react'
+import SortedTable from 'sorted-table'
 import TabButton from 'tab-button'
-import { deleteSr } from 'xo'
+import { formatSize } from 'utils'
+import { createSelector } from 'selectors'
+import { deleteSr, getUnheathlyChains } from 'xo'
 import { Container, Row, Col } from 'grid'
 
+// ===================================================================
+
+const COLUMNS = [
+  {
+    name: _('srUnHealthyVdiNameLabel'),
+    itemRenderer: vdi => <span>{vdi.name_label}</span>,
+    sortCriteria: vdi => vdi.name_label
+  },
+  {
+    name: _('srUnHealthyVdiSize'),
+    itemRenderer: vdi => formatSize(vdi.size),
+    sortCriteria: vdi => vdi.size
+  },
+  {
+    name: _('srUnHealthyVdiDepth'),
+    itemRenderer: vdi => vdi.depth,
+    sortCriteria: vdi => vdi.depth
+  }
+]
+
+class UnHealthyVdiChains extends Component {
+  _getUnHealthyVdiChains = createSelector(
+    () => this.props.sr,
+    async sr => {
+      const chains = await getUnheathlyChains(sr)
+      this.setState({
+        chains
+      })
+    }
+  )
+
+  render () {
+    const { chains } = this.state
+    this._getUnHealthyVdiChains()
+
+    return chains && chains.length !== 0
+      ? <div>
+        <h3>{_('srUnHealthyVdiTitle')}</h3>
```

Comment:
What do you think of keeping the title in the parent component and only returning the `<SortedTable ... />` here?

The big component would handle the layout of the page and this component would only handle the display of the unhealthy VDI chains. I think it would be easier to maintain and would share the problems better.